src/NLog.Extensions.Logging/Targets/MicrosoftILoggerTarget.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using NLog.Layouts;
+using NLog.Targets;
+using EventId = Microsoft.Extensions.Logging.EventId;
+
+namespace NLog.Extensions.Logging
+{
+    /// <summary>
+    /// Forwards NLog LogEvents to Microsoft ILogger-interface with support for NLog Layout-features
+    /// </summary>
+    [Target("ExtensionILogger")]
+    public class MicrosoftILoggerTarget : TargetWithContext
+    {
+        private readonly Microsoft.Extensions.Logging.ILogger _logger;
+
+        /// <summary>
+        /// EventId forwarded to ILogger
+        /// </summary>
+        public Layout EventId { get; set; }
+
+        /// <summary>
+        /// EventId-Name forwarded to ILogger
+        /// </summary>
+        public Layout EventName { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MicrosoftILoggerTarget" /> class.
+        /// </summary>
+        /// <param name="logger">Microsoft ILogger instance</param>
+        public MicrosoftILoggerTarget(Microsoft.Extensions.Logging.ILogger logger)
+        {
+            _logger = logger;
+            Layout = "${message}";
```

Comment:
Asked before, but why is the default other than other layout renderers?

Reply:
MicrosoftILoggerTarget should work as a magic converter. So when doing `nlogger.Info("Hello {planet}", "World")` then it should be like doing `ilogger.LogInformation("Hello {planet}", "World");`. Should not automatically include Level + Timestamp + etc. in the formatted message.

Reply:
Ok thanks,was also thinking something like that :)